### PR TITLE
Make it work on Android

### PR DESCRIPTION
**Pull request by cooperka**
There were several changes necessary to get react-native-snackbar working for Android. They were all fairly straightforward, and you can see the descriptions in the commits below. Please let me know if you have any questions.

It seems you also forgot to publish this project on npm. Fow now, I've published it under my own user account. If you'd like to have it back, just let me know and I'll let you publish it yourself. Cheers!

### CODE REVIEW

**Comment by Liroo** [comment]
I'm gonna try it tomorrow :) 🎸

**Comment by Liroo** [comment]
Sorry that it takes long time but I have much work side of this project @cooperka 😞

**Comment by cooperka** [comment]
No worries @Liroo! The published version on npm is working, so that's good at least :palm_tree:

**Comment by Liroo** [comment]
I had totally forgotten this repository, we don't use anymore this module. I check this now !!

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+Implementation of the material design Snackbar component for both Android and iOS. See [Google Design](https://material.google.com/components/snackbars-toasts.html) for more info.
 
 ## Installation
+
 To use this extension, you have to do the following steps:
 
-1. In your react-native project, run 
+1. In your react-native project, run
 	```
 	npm install react-native-snackbar
```
Comment (Liroo) [inline README.md:11]: prefer use `yarn add react-native-snackbar`

**Review by Liroo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Liroo) [reply README.md:11]: Both way works, and PR was submit 2 month ago. But RN community is really hooked to `yarn`.

Comment (cooperka) [reply README.md:11]: I love `yarn`! Started using it recently too.

**Comment by cooperka** [comment]
Thanks @Liroo! As you're not using this module anymore, do you plan to continue supporting this repo?

**Comment by Liroo** [comment]
Not at all, I'm gonna look at PR and issues, no more 👎 
Feel free to copy this repo and I could set a link to your repository with a note:
`This repo is no maintained anymore, follow this link ...`

**Comment by cooperka** [comment]
I see, that's probably a good idea then. I'd be happy to continue maintaining it. I currently have my fork, but it might be smoother if you simply transfer ownership of this repo? That way we can preserve PRs, issues, contributors, etc. It can be done under [settings](https://github.com/azendoo/react-native-snackbar/settings) if you like. Thoughts?

**Comment by Liroo** [comment]
@cooperka Let's do this, I transfer ownership (I'm asking to other devs if they agree)

**Comment by Liroo** [comment]
@cooperka Delete your fork that I can transfer ownership

**Comment by cooperka** [comment]
@Liroo done, it's deleted now

**Comment by cooperka** [comment]
Accepted! Thanks for helping @Liroo, I'll make sure to keep it in good shape 🍻

**Comment by Liroo** [comment]
You're welcome, I prefer mostly dev in JS than Nativ so if you want help on this repo for any JS part, ping me ;)

Keep this repo healthy @cooperka ;)

**Pull request merged**